configs/myconfigs/sampling/Sampling.py

```diff
@@ -0,0 +1,361 @@
+# -*- coding: utf-8 -*-
```

Comment:
In a way, I have pulled out parts of sample.py into the library and the rest into the example scripts.
i am planning to remove sample.py, would that be fine?